test/evergreen.yml

```diff
@@ -1048,6 +1068,9 @@ tasks:
         vars:
           test_config_filename: configs/base_test_stress.txt
           test_name: base_test
+      - func: "upload cpp test stats"
+        vars:
+          test_name: base_test
```

Comment:
Both `cppsuite-base-test-stress` and `cppsuite-hs-cleanup-default` are uploading a file called `base_test.json`. To be safe I checked that one test output doesn't overwrite the other and things are ok 👍 .

Reply:
Thanks for checking @ajmorton !